src/Winium.Desktop.Driver/CommandExecutors/NewSessionExecutor.cs

```diff
@@ -32,6 +32,17 @@ protected override string DoImpl()
             // Gives sometime to load visuals (needed only in case of slow emulation)
             Thread.Sleep(this.Automator.ActualCapabilities.LaunchDelay);
 
+            // Update running application process in Application class if it's exited base on input process name
+            if (this.Automator.Application.HasExited())
+            {
+                // Add parse process name pass from request
+                var processName = (this.ExecutedCommand.Parameters["desiredCapabilities"]["processname"]);
+                // Update launched process by process name if it's exited
+                if (processName != null)
+                {
+                    this.Automator.Application.UpdateRunApplicationProcessBy(processName.ToString());
```

Comment:
Consider the function name, it's hard to read 

Application.UpdateRunApplicationProcessBy ==> Application is duplicated 2 times for readabilty, shorter function name is better?